### DISCUSSION TITLE
preserve unicode escaped sequence

### DISCUSSION THREAD

**Discussion by Lookout-ent** [babel/babel#16115] [Q&A]
input : var El = {
    \u00C0: "A",
    \u00C1: "A",
    \u00C2: "A",
    \u00C3: "A",
    \u00C4: "A",
    \u00C5: "A",
    \u00E0: "a"}
    
```
var ast = babelParser.parse(input);

let babelgeneratorOptions = {
        compact: (options && options.compact) || false,
        concise:true,
        jsonCompatibleStrings:true,
        jsescOption : {
            quotes: 'single',
            minimal: true
        }
    }
babelGenerator(ast,babelgeneratorOptions).code;
```

this gives var El = { À: "A", Á: "A", Â: "A", Ã: "A", Ä: "A", Å: "A", à: "a"}
how do i preserve the original characters ?

**Comment by liuxingbaoyu** [comment]
Currently babel doesn't seem to be able to do it.🤔

**Reply by liuxingbaoyu** [reply]
const code = `var El = {
"\u00C0": "A",
\u00C1: "A",
\u00C2: "A",
\u00C3: "A",
\u00C4: "A",
\u00C5: "A",
\u00E0: "a"
}
`;
Maybe you could wrap the name in quotes.

**Reply by Lookout-ent** [reply]
wrapping name in quotes wouldn't fit my use-case 
i came across : https://github.com/babel/babel/discussions/13104
not able to understand how excluding @babel/plugin-transform-literals is helping

**Reply by Lookout-ent** [reply]
i would expect this to get controlled by some parser/generator options

**Comment by liuxingbaoyu** [comment]
https://babel.dev/docs/babel-plugin-transform-property-literals
You can use it.
